openqa-investigate

```diff
@@ -374,7 +374,22 @@ investigate() {
     comment_id=$(sync_via_investigation_comment "$id" "$first_cluster_job_id") || rc=$?
     [[ $rc != 255 ]] && return $rc
 
-    out=$(trigger_jobs "$id" "${@:2}") || return $?
+    out=""
+    if ! out=$(trigger_jobs "$id" "${@:2}"); then
+        first_comment_id=$(client-get-job-comments "$first_cluster_job_id" | runjq -r '[.[] | select(.text | contains("Investigation aborted"))] | min_by(.id) | .id')
+        if [[ $comment_id != "$first_comment_id" ]]; then
+            client-delete-job-comment "$first_cluster_job_id" "$comment_id"
+        fi
+        local comment="Investigation aborted for job $id.
+
+Reasons: clone failure.
+
+Suggestions: check for missing repositories or invalid configurations, investigate the job manually!"
+        client-put-job-comment "$first_cluster_job_id" "$first_comment_id" "$comment"
+        echoerr "Unable to complete the investigation on job $id due to clone failure, check for: missing repositories, invalid configurations or use the env variable 'force=true' to trigger investigation jobs"
```

Comment:
Note that the log will already contain an error message, so we probably don't need this additional one.

Reply:
Removed!

Reply:
by the way this review is in contrast to what oliver and marius comments above: https://github.com/os-autoinst/scripts/pull/398#discussion_r2073509225 and https://github.com/os-autoinst/scripts/pull/398#discussion_r2073510998

Reply:
Oh, I see. Definitely misunderstood!